tsconfig.json

```
@@ -1,9 +1,9 @@
 {
   "compilerOptions": {
     "inlineSources": false,
-    "module": "commonjs",
+    "module": "ES2020",
     "moduleResolution": "node",
-    "target": "ES2017"
+    "target": "ES2020"
```

```suggestion
    "target": "ES2019"
```

https://github.com/Borewit/strtok3/pull/533#discussion_r676503370